### PR TITLE
docs: api update example with router.START_LOCATION

### PR DESCRIPTION
**Pull request by Alex-Sokolov**

### CODE REVIEW

File: docs/api/README.md [hunk 1]
```diff
@@ -356,7 +356,7 @@ Since it's just a component, it works with `<transition>` and `<keep-alive>`. Wh
   })
 
   router.beforeEach((to, from) => {
-    if (from === START_LOCATION) {
+    if (from === router.START_LOCATION) {
```
Comment (posva) [inline docs/api/README.md]: It should be static property, so
```suggestion
    if (from === Router.START_LOCATION) {
```

Can you update zh and ru files as well in the docs folder?

**Review by posva — CHANGES_REQUESTED** [review]

File: docs/api/README.md (hunk 1 above)
Comment (Alex-Sokolov) [reply docs/api/README.md:359]: Maybe also change `Router` to `VueRouter`? Because now we have `router` and `Router` which are confusing

**Comment by Alex-Sokolov** [comment]
@posva Thanks for review! Maybe also change `Router` to `VueRouter`? Because now we have `router` and `Router` which are confusing

**Comment by posva** [comment]
You are right, that should avoid some mistakes

**Comment by Alex-Sokolov** [comment]
@posva changed too

**Review by posva — APPROVED** [review]
Thanks!

**Pull request merged**